src/java.desktop/windows/classes/sun/awt/PlatformGraphicsInfo.java

```diff
@@ -28,20 +28,45 @@
 import java.awt.GraphicsEnvironment;
 import java.awt.Toolkit;
 
+import sun.awt.windows.WToolkit;
+
 public class PlatformGraphicsInfo {
 
+    private static boolean hasDisplays;
```

Comment:
I believe it can be declared `final`, it's initialised in the static initialiser that follows the declaration.
```suggestion
    private static final boolean hasDisplays;
```

Reply:
Good catch. I updated it.